### PR TITLE
fix: task 2

### PR DESCRIPTION
**Pull request by tkachenya41**

### CODE REVIEW

File: task/02-numbers-tasks.js [hunk 1]
```diff
@@ -73,7 +71,11 @@ function getAverage(value1, value2) {
  *   (-5,0) (10,-10) => 18.027756377319946
  */
 function getDistanceBetweenPoints(x1, y1, x2, y2) {
-    throw new Error('Not implemented');
+  return Math.sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2);
+  //   return Math.sqrt(
```
Comment (ak-krivonosov) [inline task/02-numbers-tasks.js]: Please, never push comments unless they contain description about something

File: task/02-numbers-tasks.js [hunk 2]
```diff
@@ -111,7 +112,12 @@ function getLinearEquationRoot(a, b) {
  *   (0,1) (1,2)     => 0
  */
 function getAngleBetweenVectors(x1, y1, x2, y2) {
-    throw new Error('Not implemented');
+  const angleRadians = Math.atan2(y2, x2) - Math.atan2(y1, x1);
+  if (angleRadians < 0) {
```
Comment (ak-krivonosov) [inline task/02-numbers-tasks.js]: Please, use a ternary operator to make it smaller

File: task/02-numbers-tasks.js [hunk 3]
```diff
@@ -202,7 +208,15 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
```
Comment (ak-krivonosov) [inline task/02-numbers-tasks.js]: It’s not an optimal solution. You use too many and too specific conditions. Please try to find a consecutiveness. It’s solvable with 2 conditions and 1 loop

**Review by ak-krivonosov — CHANGES_REQUESTED** [review]
Left some comments

File: task/02-numbers-tasks.js (hunk 3 above)
Comment (tkachenya41) [reply task/02-numbers-tasks.js]: I had corrected but i don't think it seems to be more readable...

Comment (ak-krivonosov) [reply task/02-numbers-tasks.js]: Give one more try. You can use internet to find a smaller solution

**Review by ak-krivonosov — CHANGES_REQUESTED** [review]
1 comment

File: task/02-numbers-tasks.js [hunk 4]
```diff
@@ -201,8 +199,29 @@ function roundToPowerOfTen(num, pow) {
  *   16 => false
  *   17 => true
  */
+// function isPrime(n) {
```
Comment (ak-krivonosov) [inline task/02-numbers-tasks.js]: Please remove meaningless comments

**Review by ak-krivonosov — CHANGES_REQUESTED** [review]
1 comment

**Review by ak-krivonosov — APPROVED** [review]

**Pull request merged**